src/psyclone/psyir/nodes/loop.py

```diff
@@ -555,3 +575,21 @@ def is_unit_literal(expr):
             my_decl = DeclGen(parent, datatype="integer",
                               entity_decls=[self.variable.name])
             parent.add(my_decl)
+
+    def _halo_read_access(self, arg):
+        '''Determines whether the supplied argument has (or might have) its
+        halo data read within this loop. Returns True if it does, or if
+        it might and False if it definitely does not.
+
+        :param arg: an argument contained within this loop.
+        :type arg: :py:class:`psyclone.gocean1p0.GOKernelArgument`
```

Comment:
Please correct the type of this argument.

Reply:
Done